src/Swarm/Game/Entity.hs

```diff
@@ -604,6 +605,16 @@ deleteAll e (Inventory cs byN h) =
  where
   n = (fst <$> IM.lookup (e ^. entityHash) cs) ? 0
 
+-- | Delete entry for a certain entity from an inventory.
+deleteEntry :: Entity -> Inventory -> Inventory
```

Comment:
Adding this function was necessary because `deleteAll` keeps the entry around at `0` count, so it still shows up in the `Installed` list in the left pane.